src/App.tsx

```diff
@@ -1,61 +1,118 @@
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 import './App.scss';
+import { getUserById } from './services/user';
+import { TodoWithUser } from './types/types';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId)
+  || {
+    id: 0, name: '', username: '', email: '',
+  },
+})) as TodoWithUser[];
```

Comment:
it's redundant let's just return null

Reply:
Not fixed 
![2023-09-05_12h00_54](https://github.com/mate-academy/react_add-todo-form/assets/61904995/2d001ab1-ced2-4e52-bf8c-e25c0d80b960)
```suggestion
  user: getUserById(todo.userId)
}));
```